modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,197 @@
+
+=== Stateful sets directories
+
+Each stateful set uses different directories to hold its data, resulting in the following directories:
+
+```
+  /var/lib/data/influxdb/data
+  /var/lib/data/influxdb/meta
+  /var/lib/data/logstash
+  /var/lib/data/dias-cassandra
+  /var/lib/data/dias-meld
+```
+
+InfluxDB meta pods (3 pods) use `/var/lib/data/influxdb/meta` while data pods (2 pods) use `/var/lib/data/influxdb/data`. Logstash uses `/var/lib/data/logstash` and finally Insight uses the remaining two directories to hold its data.
+
+== Requirements
+
+Hardware requirements are listed in the main PCE hardware section but 3 nodes are required for all Anypoint Monitoring/Visualizer workload.
+
+Port 8895 needs to be accesible in the main balancer along with the rest of the required ports.
+
+
+=== InfluxDB License
+
+InfluxDB requires a valid license to operate, which should have been provided at the time of purchase of Anypoint Monitoring.
+
+Anypoint Monitoring requires a Kubernetes secret to be present in the cluster before all Anypoint Monitoring/Visualizer charts are deployed.
+
+The following command needs to be run in order to create the required secret in the default namespace:
+
+
+```
+kubectl create secret generic influxdb-license --from-literal=license={license}
+```
+
+Note that InfluxDB license might be a JSON file in which case we only need the `key` value in that secret.
+
+If no secret is given, InfluxDB will fail with the following message
+
+```
+ts=ZULU_TIME lvl=error msg="LICENSE IS INVALID OR CANNOT BE FOUND" log_id=LOG_ID service=licensing error=ERROR_MESSAGE
+```
+
+If that is the case, the secret will need to be created and InfluxDB chart redeployed. Before redeploying InfluxDB, all *pvc*, *pv* and *jobs* (named `dias-prov-k8s-am-influxdb-comp-bootstrap`) will need to be deleted.
+
+
+
+== Restore/Backup
+
+Both backup and restore procedures for Anypoint Monitoring are outside the normal Gravitational backup/restore mechanism and therefore requires additional procedures.
+
+Anypoint Monitoring/Visualizer need to backup the following component's data
+
+. InfluxDB (`dias-prov-k8s-am-influxdb-comp`)
+. Cassandra (`dias-prov-k8s-am-ingestor-comp`)
+
+
+Each component has its own way to backup, and restore, the data. It is important to mentioned that the Anypoint Monitoring/Visualizer custom Stolon DB will be backup/restore by the default Gravitational mechanism and it is not considered in this document.
```

Comment:
This is a good clarification. Instead of calling it custom Stolon DB, let's call it postgres database that holds configuration for the Monitoring and Visualizer components.

Reply:
Added.